src/helpers/getStatusBarText/index.ts

```diff
@@ -1,11 +1,24 @@
+import { workspace, WorkspaceConfiguration } from 'vscode'
 import getFormattingStatus from '../getFormattingStatus'
 
-export const ENABLED_TEXT = 'Formatting: $(check)'
-export const DISABLED_TEXT = 'Formatting: $(x)'
+type StatusBarLabelT = {
+  formattingEnabled: string
+  formattingDisabled: string
+};
```

Comment:
Interesting, I’d have expected VSCode to automatically generate types :D

Reply:
It may be my VSCode setup that was giving me false-positives with these. I'm happy to remove entirely :)